docs_src/microservices/device/supported/device-usb-camera/walkthrough/general-usage.md

```diff
@@ -23,6 +26,23 @@ Query parameter:
     }' http://localhost:59882/api/{{api_version}}/device/name/<device name>/StartStreaming
     ```
 
+!!! note
+    If running in secure mode all the api executions (for this api and subsequent apis) need the JWT token generated previously. E.g.
+    ```bash
+    curl -X PUT -d '{
+        "StartStreaming": {
+        "InputImageSize": "640x480",
+        "OutputVideoQuality": "5"
+        }
+    }' http://localhost:59882/api/{{api_version}}/device/name/<device name>/StartStreaming \
+    --header 'Authorization: Bearer <jwt-token>'
+    ```
+
+The response to the above call should look similar to the following:
```

Comment:
I think in other parts of the documentation we say "Example  Output" or "Good Response"

Reply:
fixed